src/meta/KVBasedGflagsManager.cpp

```diff
@@ -0,0 +1,141 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "meta/KVBasedGflagsManager.h"
+#include "meta/processors/Common.h"
+#include "meta/MetaServiceUtils.h"
+
+DECLARE_int32(load_config_interval_secs);
+
+namespace nebula {
+namespace meta {
+
+KVBasedGflagsManager::KVBasedGflagsManager(kvstore::KVStore* kv) {
+    kvstore_ = dynamic_cast<kvstore::NebulaStore*>(kv);
+    CHECK_NOTNULL(kvstore_);
+}
+
+KVBasedGflagsManager::~KVBasedGflagsManager() {
+    kvstore_ = nullptr;
+}
+
+Status KVBasedGflagsManager::init() {
+    getGflagsModule();
+    declareGflags();
+    return registerGflags();
+}
+
+folly::Future<StatusOr<bool>>
+KVBasedGflagsManager::setConfig(const cpp2::ConfigModule& module, const std::string& name,
+                                const cpp2::ConfigType& type, const VariantType &value) {
+    UNUSED(module); UNUSED(name); UNUSED(type); UNUSED(value);
+    LOG(FATAL) << "Unimplement!";
+    return Status::OK();
+}
+
+folly::Future<StatusOr<std::vector<cpp2::ConfigItem>>>
+KVBasedGflagsManager::getConfig(const cpp2::ConfigModule& module, const std::string& name) {
+    UNUSED(module); UNUSED(name);
+    LOG(FATAL) << "Unimplement!";
+    return Status::OK();
+}
+
+folly::Future<StatusOr<std::vector<cpp2::ConfigItem>>>
+KVBasedGflagsManager::listConfigs(const cpp2::ConfigModule& module) {
+    UNUSED(module);
+    LOG(FATAL) << "Unimplement!";
+    return Status::OK();
```

Comment:
Although LOG(FATAL) will break down the programming, it's seems using `Status::NotSupported()` to replace `Status::OK()` would be reasonable. just personal understanding. :)

Reply:
Sure, sounds reasonable